schemas/get-worker-type-response.yml

```diff
@@ -0,0 +1,112 @@
+id:       http://schemas.taskcluster.net/aws-provisioner/v1/get-worker-type-response.json#
```

Comment:
Technically you don't need the `workerType` property in either... But it's nice to have... so I'll leave that decision to you... Making them use the same schema is neat though, it reduces bugs :)